tensorflow/lite/kernels/sub_test.cc

```diff
@@ -333,8 +333,8 @@ void QuantizedTestsActivationRELU_N1_TO_1() {
   std::vector<std::vector<float>> results = {{-1.0, -0.2, 0.0, 1.0},
                                              {-1.0, -0.2, 1.0, 0.2}};
   for (int i = 0; i < inputs1.size(); ++i) {
-    QuantizedSubOpModel m({tensor_type, {1, 2, 2, 1}, -1.0, 1.0},
-                          {tensor_type, {1, 2, 2, 1}, -1.0, 1.0},
+    QuantizedSubOpModel m({tensor_type, {1, 2, 2, 1}, -0.9, 0.9},
```

Comment:
why change this?

Reply:
While changing the SUB kernel to avoid negative quantized multipliers an error slipped by as the quantized SUB kernel tests were not properly testing the operator. All the tests had the same scaling/zero-point for both inputs and the output which mean that a quantized kernel that just do a subtraction without taking into account the scaling would work. I fixed the int16 SUB kernel in 79c6ccc and adapted the tests in 2e94bea so that such errors can be catch.